src/hooks/useUniswapXSwapCallback.ts

```diff
@@ -80,13 +80,16 @@ export function useUniswapXSwapCallback({
           }
         }
 
+        const beforeSign = Date.now()
         const { signature, updatedOrder } = await signDutchOrder()
 
         sendAnalyticsEvent(SwapEventName.SWAP_SIGNED, {
           ...formatSwapSignedAnalyticsEventProperties({
             trade,
             allowedSlippage,
             fiatValues,
+            // measures the amount of time the user took to sign the permit message in their wallet
```

Comment:
nit: comment should go in the central `formatSwapSignedAnalyticsEventProperties`